de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/ISarosSession.java

```diff
@@ -386,6 +388,12 @@ public void addSharedResources(
    */
   public void changeColor(int colorID);
 
+  public void gitSendCommitRequest();
```

Comment:
The ISarosSession interface is already in a bad state. There is no reason to pollute it further.

Reply:
done